examples/jsm/tsl/shadows/TileShadowNode.js

```diff
@@ -279,11 +279,22 @@ class TileShadowNode extends ShadowBaseNode {
 		scene.overrideMaterial = getShadowMaterial( light );
 		renderer.setRenderTarget( this.shadowMap );
 
+		const cameraLayers = [];
```

Comment:
This array object should not be created per update step. I suggest you move it into module scope similar to the arrays we use for chain maps.

Reply:
Done!